src/handedness.rs

```diff
@@ -0,0 +1,32 @@
+use std::fmt::Debug;
+
+pub trait Handedness: Clone + Copy + Debug {
```

Comment:
If you include `static here, it doesn't need to be scattered around

```suggestion
pub trait Handedness: Clone + Copy + Debug + 'static {
```